src/main/resources/static/js/endpoints.js

```diff
@@ -288,4 +289,19 @@ export function designSpaceMerge(inputSpaces, outputSpace, tolerance){
   } else {
     swalError(request.response);
   }
+}
+
+export function designSpaceReverse(inputSpaces, outputSpace, tolerance){
+    let query = "?";
+    query += encodeQueryParameter("inputSpaceID", inputSpaces, query);
```

Comment:
I'd prefer that we keep the plurality of the nouns constant. The user should only be able to enter one inputSpace, so this variable name is misleading and confusing.

Reply:
I will change it.